cmd/trustlines.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// trustlinesCmd represents the trustlines command
+var trustlinesCmd = &cobra.Command{
+	Use:   "export_trustlines",
+	Short: "Exports the trustline data over a specified range.",
+	Long:  `Exports the trustline data over a specified range to an output file`,
```

Comment:
Why is this string in ``, while the previous is in double-quotes?

Reply:
The `` are for multi-line strings. Right now it doesn't matter, but in the future I thought we might flesh out the help message, which could extend it across multiple lines.